testsuite/forge-cli/src/main.rs

```diff
@@ -2400,7 +2427,7 @@ fn indexer_test() -> ForgeConfig {
         (
             TransactionWorkload::new(TransactionTypeArg::TokenV2AmbassadorMint, 20000)
                 .with_unique_senders(),
-            (3200, 0.5, 0.5, 0.5),
+            (2000, 0.5, 0.5, 0.5),
```

Comment:
If I want to run an adhoc forge with only one of the workloads, would I need to create a PR commenting out the other workloads and run the adhoc from that PR?

Reply:
yea, or the better way is to create (in a separate PR) a testsuite for each that you want to run independently. So instead of using the `indexer_test` suite maybe you have like `indexer_{p2p, graffio, ...}`